### PR TITLE
Feature/remove old python documents and add roles

### PR DESCRIPTION
**Pull request by kjam**
Hi folks,

I could definitely use a second set of eyes on these changes. Please feel free to suggest here anything that needs reworded or is confusing. My goal was to remove some old documentation that is no longer relevant and to begin adding more documentation for Cape Cloud, pycape and how the platform works.

Thanks for your help!

-kj

### CODE REVIEW

File: docs/libraries/index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Cape Privacy provides libraries that allow you to:
 
-* Train models in collaboration with other entities without revealing the contents of your datasets.
-* Write data security policies in YAML format.
-* Create scripts that apply your policies to your data.
-* Use Cape Privacy's standard data transformations. <!-- TODO: coming soon: can also write your own -->
\ No newline at end of file
+* Train models in collaboration with other organizations without revealing the contents of your datasets.
+* Get started using encrypted learning without having to learn cryptography.
```
Comment (gifteconomist) [inline docs/libraries/index.md:6]: 💯

File: docs/understand/features/roles.md [hunk 2]
```diff
@@ -1,22 +1,67 @@
...
+Organizational Administrator Actions:
 
- * `project-reader`: a project-reader can see what project details (for example, the title and description), and read what the policy is.
- * `project-contributor`: a project-contributor can do everything a reader can, plus suggest policy changes.
- * `project-owner`: a project-owner can do everything a contributor can do, plus approve or reject policy suggestions, add users to the project, and so on.
+- Full permissions for organization
+- Can add or remove persons from the organization
+- Can change roles for any members of the organization
+- Can delete the organization
+- Full permissions for all projects that organization is a part of
```
Comment (gifteconomist) [inline docs/understand/features/roles.md:18]: All of the permissions below this line feels like it falls into this category of "full project permissions" (Except maybe `Can join or leave projects on behalf of an organization`, this feels like a org-level permission). So many we can add a colon/sublist here like: 

> Full permissions for all projects that organization is a part of, including: 
> - Can add or edit contributors on all projects
> - Can add, remove data views for all projects
> - Can approve and run encrypted learning jobs for all projects

File: docs/understand/features/roles.md [hunk 3]
```diff
@@ -1,22 +1,67 @@
...
 
-!!! note
-    Cape refers to all users associated with a project as "contributors" to that project, and manages project users with the `contributors` command. In the documentation, "contributor" refers to any user in the project, while `project-contributor` refers to a user with the `project-contributor` role described in this document.
+## Project Administrator
+
+A project administrator has full permissions for their organization's projects on Cape Privacy, including the ability to join or leave projects on behalf of an organization, add and edit contributors on all projects and perform necessary project actions like adding and removing data views or approving and running encrypted learning jobs.
+
+Project Administrator Actions:
+
+- Full permissions for all projects that organization is a part of
```
Comment (gifteconomist) [inline docs/understand/features/roles.md:31]: Same "full-permissions" nesting suggestion here.

File: docs/understand/features/roles.md [hunk 4]
```diff
@@ -1,22 +1,67 @@
...
+- Can add or edit contributors on all projects
+- Can add, remove data views for all projects
+- Can approve and run encrypted learning jobs for all projects
 
 
-!!! note
-    Cape refers to all users associated with a project as "contributors" to that project, and manages project users with the `contributors` command. In the documentation, "contributor" refers to any user in the project, while `project-contributor` refers to a user with the `project-contributor` role described in this document.
+## Project Administrator
+
+A project administrator has full permissions for their organization's projects on Cape Privacy, including the ability to join or leave projects on behalf of an organization, add and edit contributors on all projects and perform necessary project actions like adding and removing data views or approving and running encrypted learning jobs.
```
Comment (gifteconomist) [inline docs/understand/features/roles.md:27]: So does the project administer have all the permissions, including those granted to "operator" and "data scientist"?

File: docs/understand/features/roles.md (hunk 2 above)
Comment (kjam) [reply docs/understand/features/roles.md:18]: Great suggestion! Thank you!

File: docs/understand/features/roles.md (hunk 4 above)
Comment (kjam) [reply docs/understand/features/roles.md:27]: They are not expected to manage organizational tokens, but otherwise, yes! Let me know if this makes sense.

Comment (gifteconomist) [reply docs/understand/features/roles.md:28]: Cool gotcha!

**Review by gifteconomist — APPROVED** [review]
Super!

**Pull request merged**